src/pages/index.js

```diff
@@ -1,12 +1,13 @@
 import "./index.css";
-
-import { initialCards, validationSettings } from "../utils/constants.js";
+import Api from "../components/Api.js";
+import { validationSettings } from "../utils/constants.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import Section from "../components/Section.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
```

Comment:
Please, delete ` this._popupForm.reset();` from line 24 in `PopupWithForm`

* You need to **reset** the form only   in block `then` in order not to do that immediately after `submit` because when there is an error, the modal will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more.